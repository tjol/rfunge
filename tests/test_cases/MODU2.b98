"UDOM"4#v(a3      v
        >a"gnissim UDOM"ck,@                                                        >'>'T4p a03-v
                                                                                  
           vp21p20<                                                                             < <
>a,       ;> 12g02g01p11pv >a,       ;> 12g02g01p11pv >a,       ;> 12g02g01p11pv;a, ^'v'T4p 0a-3^
 v.:M,k4"M -> ".:g10.:g11<  v.:U,k4"U -> ".:g10.:g11<  v.:R,k4"R -> ".:g10.:g11<    >'@'T4p 0a-03-^
 >a, 11p                 v  >a, 11p                 v  >a, 11p                 v
 v.:M,k4"M -> ".:g10.:g11<  v.:U,k4"U -> ".:g10.:g11<  v.:R,k4"R -> ".:g10.:g11<
 >a, 01g+11p             v  >a, 01g+11p             v  >a, 01g+11p             v
^ .:M,k4"M -> ".:g10.:g11< ^ .:U,k4"U -> ".:g10.:g11< ^ .:R,k4"R -> ".:g10.:g11<
