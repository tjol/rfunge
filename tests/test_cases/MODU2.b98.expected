10 3 M -> 1 
1 3 M -> 1 
4 3 M -> 1 
10 3 U -> 1 
1 3 U -> 1 
4 3 U -> 1 
10 3 R -> 1 
1 3 R -> 1 
4 3 R -> 1 

10 -3 M -> -2 
-2 -3 M -> -2 
-5 -3 M -> -2 
10 -3 U -> 1 
1 -3 U -> 1 
-2 -3 U -> 1 
10 -3 R -> 1 
1 -3 R -> 1 
-2 -3 R -> -2 

-10 3 M -> 2 
2 3 M -> 2 
5 3 M -> 2 
-10 3 U -> 2 
2 3 U -> 2 
5 3 U -> 2 
-10 3 R -> -1 
-1 3 R -> -1 
2 3 R -> 2 

-10 -3 M -> -1 
-1 -3 M -> -1 
-4 -3 M -> -1 
-10 -3 U -> 2 
2 -3 U -> 2 
-1 -3 U -> 2 
-10 -3 R -> -1 
-1 -3 R -> -1 
-4 -3 R -> -1 

